visualization/motion_planning_rviz_plugin/src/motion_planning_frame_planning.cpp

```diff
@@ -129,16 +137,48 @@ void MotionPlanningFrame::computePlanButtonClicked()
 void MotionPlanningFrame::computeExecuteButtonClicked()
 {
   if (move_group_ && current_plan_)
-    move_group_->execute(*current_plan_);
+  {
+    // ui_->stop_button->setEnabled(true); // enable stopping
```

Comment:
Because it's currently not working. A service cannot be interrupted. @wkentaro suggested an option which I will try later.